Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -2077,37 +2041,26 @@ namespace OpenInfraPlatform {
 				}
 
 				/**********************************************************************************************/
-				/*! \brief Calculates a trimming point on the circle using \c IfcCartesianPoint.
-				* \param[in] circle					A pointer to data from a \c IfcCircle.
-				* \param[in] cartesianPoint			A pointer to data form \c IfcCartesianPoint.
+				/*! \brief Calculates a trimming point on the curve using \c IfcCartesianPoint.
+				* \tparam curve						A pointer to data from one of curves.

```

Comment:
the template parameter is called `TCurve`

Reply:
Resolved: 2816eed